src/api/store/graph.py

```diff
@@ -393,3 +405,44 @@ def list_graphs_for_super_admin(self, context: Context):
     except Exception as e:
       capture_message(str(e), level="error")
       return None, CustomMassenergizeError(str(e))
+
+  def debug_data_fix(self) -> None:
+    # attempting to fix the problem with data getting screwed up
+    for community in Community.objects.all().select_related('goal'):
+      if community.goal:
+        action_goal= max(community.goal.target_number_of_actions, 100)
+      else:
+        # communities that don't have goals
+        action_goal = 100
+
+      if community.is_geographically_focused:
+        user_actions = UserActionRel.objects.filter(
+          real_estate_unit__community=community, status="DONE"
+        )
+      else:
+        user_actions = UserActionRel.objects.filter(
+          action__community=community, status="DONE"
+        )
+
+
+      for d in Data.objects.filter(community=community):
+
+      #for tag in TagCollection.objects.get(name__icontains="Category").tag_set.all():
+        #d = Data.objects.filter(community=community, name=tag.name).first()
+        if d and d.value>action_goal:
+          oldval = d.value
+          val = 0
+          tag = d.tag
+
+          for user_action in user_actions:
+            if user_action.action and user_action.action.tags.filter(pk=tag.id).exists():
+              val += 1
+
+          if (val != d.value) :
+            d.value = val
+            d.save()
+            print("WARNING - data_fix: Community: " + community.name
```

Comment:
maybe we don't want to commit print statements as they will be dumped in AWS logs which is virtually inaccessible to us.